src/core/modules/StarMgr.cpp

```diff
@@ -751,7 +751,7 @@ auto StarMgr::loadCommonNames(const QString& commonNameFile) const -> CommonName
 #if QT_VERSION >= QT_VERSION_CHECK(6,0,0)
 			StarId hip = recMatch.captured(1).toULongLong(&ok);
 #else
-			StarId hip = recMatch.captured(1).toInt(&ok);
+			StarId hip = recMatch.captured(1).toULongLong(&ok);
```

Comment:
Ifdef for Qt is not needed here

Reply:
I wondered why @10110111 used the fallback to Int here and suspected a problem with Qt5.12 which I no longer have. If CI is happy, this of course goes away.

Reply:
I think it was just a change by inertia when I was changing the conversion of a `QJsonValue`. Of course, `QString` doesn't have this problem.